Measurements/Tests.qs

```diff
@@ -281,7 +281,7 @@ namespace Quantum.Kata.Measurements {
             [BoolArrayAsKetState(b1), BoolArrayAsKetState(b2)]);
     }
 
-    @Test("Microsoft.Quantum.Katas.CounterSimulator")
+    @Test("QuantumSimulator")
```

Comment:
Similarly here, I'm not sure all multi-qubit tasks should be switched to `QuantumSimulator`, since they call `DistinguishStates_MultiQubit` which uses `GetOracleCallsCount` if `measurementsPerRun > 0`. In particular, tasks 1.7 and 1.8 have a restriction on the number of measurements the solution is allowed to do. CI build won't catch that, since it only validates that correct solutions pass, not that incorrect solutions fail. Have you checked that doing extra measurements in the solution fails the tests?

Reply:
Here I get the same as above, when adding two measurements to task 1.7 and 1.8 I get:
```
You are allowed to do at most one measurement, and you did 2
	Expected:	True
	Actual:	False
Try again!
```

I could still bring back the `CounterSimulator` to be sure.